src/components/form/Form/Form.tsx

```diff
@@ -86,33 +108,35 @@ interface FormProps extends Omit<StrictFormProps, 'error'> {
  *
  */
 
-export default function Form({
+function Form({
     children,
     errors,
     errorMessageHeader = 'Errors in the form',
     onErrorsDismiss = () => {},
     scrollToError = false,
     ...formProps
-}: PropsWithChildren<FormProps>): ReactElement | null {
+}: FormProps): ReactElement | null {
     const formRef = useRef<HTMLElement>(null);
     useEffect(() => {
         if (scrollToError && !_.isEmpty(errors)) {
             formRef.current?.scrollIntoView?.({ behavior: 'smooth', block: 'start' });
         }
     }, [errors]);
 
-    let formattedErrors = errors;
+    let formattedErrors: undefined | Array<string> | ReactElement<unknown, string | JSXElementConstructor<any>>;
```

Comment:
```suggestion
    let formattedErrors: undefined | Array<string> | JSX.Element;
```
Would be shorter and in line with `errors` type.